extras/videoDrivers/SDL/VideoSDL.cpp

```diff
@@ -14,21 +14,30 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Why do we require this? Makes the license header even longer :/

Reply:
see https://spdx.org/ids-how

Reply:
Ok that is how, but why? The license texts are already "standard"/fixed-and-known. So what benefit do we get for adding these 2 lines to 3k files?

Reply:
Just seen that Linux 5.2 completely replace the license boilerplate with this 1 liner. See https://www.kernel.org/doc/html/latest/process/license-rules.html for rules and https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/diff/arch/arm/boot/dts/hi3519.dtsi?id=2c1212de6f9794a7becba5f219fa6ce8a8222c90 for an example. Not sure if we need the `Copyright...` line (e.g. https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/alpha/include/asm/xor.h?id=86c2f5d653058798703549e1be39a819fcac0d5d does not have it), so I'd at least remove all license text, have the SPDX as first line and potentially also get rid of the Copyright lines if they don't provide any actual benefit. If they do, keep them as a 2nd one-liner.